openedx/core/djangoapps/discussions/models.py

```diff
@@ -159,6 +174,23 @@ class Features(Enum):
             '',
             '',
         )._asdict(),
+        "messages": ProviderMessages(
```

Comment:
While you are updating `ProviderExternalLinks` to have named arguments I'd suggest to update every `ProviderExternalLinks` in this file to keep the changes consistent.